tests/webserver/conftest.py

```diff
@@ -0,0 +1,97 @@
+#  This file is part of the QuestionPy Server. (https://questionpy.org)
+#  The QuestionPy Server is free software released under terms of the MIT license. See LICENSE.md.
+#  (c) Technische Universität Berlin, innoCampus <info@isis.tu-berlin.de>
+
+import asyncio
+import threading
+from pathlib import Path
+from typing import Generator, Callable
+from zipfile import ZipFile
+
+import pytest
+import yaml
+from aiohttp import web
+from click.testing import CliRunner
+from questionpy_common.manifest import Manifest
+from selenium import webdriver
+
+from questionpy_sdk.commands.package import package
+from questionpy_sdk.resources import EXAMPLE_PACKAGE
+from questionpy_sdk.webserver.app import WebServer
+
+
+@pytest.fixture
+def package_and_manifest() -> Generator[tuple[Path, Manifest], None, None]:
```

Comment:
Hier und weiter unten wäre der `Iterator`-Typ aus `collections.abc` m.M.n. passender.

```suggestion
def package_and_manifest() -> Iterator[tuple[Path, Manifest]]:
```